addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -208,7 +232,7 @@ def _l10n_es_edi_facturae_convert_payment_terms_to_installments(self):
                 installments.append({
                     'InstallmentDueDate': payment_term.date_maturity,
                     'InstallmentAmount': payment_term.amount_residual_currency,
-                    'PaymentMeans': '04',  # Credit Transfer
+                    'PaymentMeans': self.l10n_es_payment_means or '04',
```

Comment:
```suggestion
                    'PaymentMeans': self.l10n_es_payment_means,
```
It makes more sense to set a default value on the field directly.
Also, please update the doc string as well, to reflect the fact that it is no longer hardcoded :wink: